pubspec.yaml

```diff
@@ -22,22 +22,29 @@ environment:
 
 dependencies:
   cupertino_icons: ^1.0.2
+  dio: ^4.0.0
   flutter:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
   flutter_config: ^2.0.0
+  freezed: ^0.14.1+3
```

Comment:
@luongvo please move `freezed` to `dev_dependencies` section by following [this tutorial](https://pub.dev/packages/freezed#how-to-use) 😄

Reply:
Updated